sched/clustermembership/cluster.go

```diff
@@ -3,8 +3,37 @@ package cluster_membership
 //go:generate mockgen -source=cluster.go -package=cluster_membership -destination=cluster_mock.go
 
 type Cluster interface {
-	/*
-	 * Returns a Slice of Node Ids in the Cluster
-	 */
+	//
+	// Returns a Slice of Node in the Cluster
+	//
 	Members() []Node
 }
+
+type DynamicCluster interface {
```

Comment:
I mean sure but any implementing class has to implement all of them.  I'll split them out, because I don't want to go back and forth, but I think this is a rather picky comment.  Sometimes SOLID gets in the way of clean simple interfaces.